server/sonar-web/src/main/js/helpers/periods.js

```diff
@@ -51,7 +50,7 @@ export function getPeriodDate(period) {
     return null;
   }
 
-  return moment(period.date).toDate();
+  return new Date(period.date);
```

Comment:
Ok it's just too messy to create a custom parsing function, I'm not confident with all that timezone stuff. So I think the easiest is to use a the parsing function of a lib (date-fns) I still need to check the size of the bundles generated by webpack to be sure its ok.
If it's not we will have to write our own but at least as it now we wont have to replace the parsing function everywhere.